tests/integration/test_data_row_media_attributes.py

```diff
@@ -5,6 +5,8 @@ def test_export_empty_media_attributes(configured_project_with_label):
     project, _, _, _ = configured_project_with_label
     # Wait for exporter to retrieve latest labels
     sleep(10)
-    labels = project.label_generator()
-    label = next(labels)
-    assert label.data.media_attributes == {}
+    labels = project.label_generator().as_list()
```

Comment:
you can also just call list(project.label_generator()). It makes it more readable.

Reply:
or even 
```
labels = [project.label_generator().get_one()]
```

to be consistent with the other test

Reply:
ah, len([None]) == 1, forget my comment )